Jenkinsfile

```diff
@@ -6,7 +6,7 @@ dockerfile {
     usePackages = true
     dockerRepos = ['confluentinc/cp-base-new', 'confluentinc/cp-jmxterm']
     slackChannel = '#tools-notifications'
-    mvnSkipDeploy = true
+    mvnSkipDeploy = false
```

Comment:
Whats the nature of this change?

Reply:
After migrating https://github.com/confluentinc/cc-docker-base/tree/master/disk-usage-agent to `common-docker`, we want to deprecate the build of `disk-usage-agent` from `cc-docker-base` and add it as a dependency that can be downloaded from Artifactory.

My understanding is that with this change, `disk-usage-agent` will be published to Artifactory...if this is not the right/best way to do this (as I am still figuring things out with maven), please let me know.

Reply:
Forgive me, I miss read the variable and incorrectly assumed we were disabling the deploy component. LGTM then.